blog/content/blog/cncf-dan.md

```diff
@@ -1,74 +1,196 @@
+status of conformance, to a multi-faceted tool with multiple ways to view not
+just the current data, but the historical progress of Conformance. Hippie
+developed a unique Kind+APISnoop combination that allowed anyone to access the
+querying power of APISnoop locally.
+
+![1_15Cover](/images/blog_image/1_15Cover.png "1.15 Cover") Test Cover 1.15
+
+![1_21Cover](/images/blog_image/1_21cover.png "1.21 Cover") Test cover 1.21
 
 ![alt_text](/images/blog_image/Conformance-progress.png "conformance-progress")
 Conformance Progress graph
 
-All the added conformance test cover was great news and added the expected prestige and value of the Certified Kubernetes brand. At the same time, it was making life increasingly difficult for [Taylor Waggoner](https://github.com/taylorwaggoner) at the CNCF managing the Conformance Certification process. 
-
-All pull requests for certification had to be manually verified to ensure it contained the correct data, but the list of conformance tests kept growing with each release. Dan approached Hippie and requested that the process should be automated, and in 2020 the CNCF CI Bot was created by [Berno Kleinhans ](https://github.com/bernokl)and [Rob Kielty](https://github.com/RobertKielty). This allowed for the automatic checking and labeling of conformance pull requests, speeding up the process and making the human effort required less. 
-
-The Kubernetes conformance journey up to this day was an eventful one, built around relationships and community cooperation, with many different contributors playing their part to help move this forward. All of this was made possible by the extraordinary vision and leadership of a friend dear to everyone in the Kubernetes community, Dan Kohn.
+The increased conformance coverage was great news, and added the expected value
+to the Certified Kubernetes brand. At the same time, it was making life
+increasingly difficult for [Taylor Waggoner](https://github.com/taylorwaggoner)
+at the CNCF to manage the Conformance Certification process. All pull requests
+for certification had to be manually verified to ensure it contained all tests,
+but the list of conformance tests kept growing with each release.
+
+Dan approached Hippie and requested the process to be automated, and in 2020 the
+CNCF CI Bot was created by [Berno Kleinhans ](https://github.com/bernokl)and
+[Rob Kielty](https://github.com/RobertKielty). This allowed for the automatic
+checking and labeling of conformance pull requests, speeding up the process and
+reducing the human effort needed.
+
+## Thank you
+
+The Kubernetes Conformance journey up to this day is an eventful one, built
+around relationships and community cooperation, with many different contributors
+playing their part to help move this forward. All of this was made possible by
+the extraordinary vision and leadership of a friend dear to everyone in the
+Kubernetes community: Dan Kohn.
```

Comment:
This is wonderful, absolutely wonderful editing Zach.
You have a gift.

Reply:
great rewrite!!